src/units/helper.hpp

```diff
@@ -49,4 +50,55 @@ std::string unit_level_tooltip(const unit &u);
  * advance to.
  */
 std::string unit_level_tooltip(const unit_type &u);
+
+/**
+ * @return If active, do nothing. If inactive, surround with pango tag that
+ * changes font color to grey.
+ */
+std::string maybe_inactive(const std::string& str, bool active);
+
+/**
+ * @return If unit recall cost is greater than team's recall cost,
+ * return a help markup string that shows the unit recall cost
```

Comment:
Is there a distinction between "help markup string" and "pango tag"?

Reply:
there is, since the `<img/>` tag is involved. It is one of the tags that is supported by rich label/help markup but not by pango.